bin/config.yml

```diff
@@ -1,4 +1,5 @@
 ---
 #jobdir: ../jobs/
 jobdir: /home/andrew/adminware/jobs/
```

Comment:
Ideally drop the /home/andrew from these default settings, eg
```
jobdir: jobs/
logfile: logs/adminware.log
statefile: var/state.yaml

```

Reply:
Made the config settings more generic now